kong/timing/hooks/dns.lua

```diff
@@ -30,6 +23,7 @@ function _M.register_hooks(timing_module)
     Here is the signature of the `toip()` function:
     function toip(self, qname, port, dnsCacheOnly, try_list)
   --]]
+  local client = assert(package.loaded["kong.resty.dns.client"])
```

Comment:
We can't use `kong.dns` here, because this `timing/hooks/dns.lua` module is loaded in `globalpatches`, which is ahead of `kong.dns` initialization.

Reply:
got it